neonvm/controllers/virtualmachine_controller.go

```diff
@@ -235,20 +235,6 @@ func (r *VirtualMachineReconciler) doFinalizerOperationsForVirtualMachine(ctx co
 	}
 }
 
-func runnerSupportsCgroup(pod *corev1.Pod) bool {
-	val, ok := pod.Labels[vmv1.RunnerPodVersionLabel]
-	if !ok {
-		return false
-	}
-
-	uintVal, err := strconv.ParseUint(val, 10, 32)
-	if err != nil {
-		return false
-	}
-
-	return api.RunnerProtoVersion(uintVal).SupportsCgroupFractionalCPU()
```

Comment:
does it make sense to do either of those?
1. Keep this line as an assert
2. Delete SupportsCgroupFractionalCPU function

Reply:
Makes sense, yeah. Added some assertions, basically. LMK what you think.